TheFourFifths/consus-client
# Delete a model and its items

### Summary

You can delete a model by clicking on the trashcan for a model in the view all models page.

### Checklist


Finally, is it ready to be reviewed and merged: yes :white_check_mark:

### How to Review

1. Start the server on delete-model-tff110 (or story if already merged)
2. View all models and click the trashcan.
3. Verify model was deleted properly
4. View all items take note of how many are left
5. Delete the last model in the system. Verify it was deleted properly
6. View all items and verify there are none left. 

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-110](https://msoese.atlassian.net/browse/TFF-110)
